app/src/main/java/com/cornellappdev/transit/ui/viewmodels/FavoritesViewModel.kt

```diff
@@ -25,7 +26,7 @@ class FavoritesViewModel @Inject constructor(
      * A flow emitting all the locations and whether or not they have been favorited.
      */
     private val favoritesFlow = userPreferenceRepository.favoritesFlow
-
+    //private val favoritesFlow: StateFlow<Set<String>> = MutableStateFlow(setOf("Gates Hall", "Duffield")).asStateFlow()
```

Comment:
any reason this is here?

Reply:
I used it to test the bottomsheet since the backend doesn't have data yet. I thought I'd keep it there since I'll have to keep using it in the future and I'm currently working on a somewhat similar feature.